### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.keyvault.models.key_vault_error.KeyVaultError
  - azure.keyvault.models.key_vault_error.KeyVaultErrorException
  fullName: azure.keyvault.models.key_vault_error
  langs:
  - python
  module: azure.keyvault.models.key_vault_error
  name: key_vault_error
  source:
    id: key_vault_error
    path: azure-keyvault\azure\keyvault\models\key_vault_error.py
    remote:
      branch: master
      path: azure-keyvault\azure\keyvault\models\key_vault_error.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.keyvault.models.key_vault_error
references:
- fullName: azure.keyvault.models.key_vault_error.KeyVaultError
  isExternal: false
  name: KeyVaultError
  parent: azure.keyvault.models.key_vault_error
  uid: azure.keyvault.models.key_vault_error.KeyVaultError
- fullName: azure.keyvault.models.key_vault_error.KeyVaultErrorException
  isExternal: false
  name: KeyVaultErrorException
  parent: azure.keyvault.models.key_vault_error
  uid: azure.keyvault.models.key_vault_error.KeyVaultErrorException
